src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -680,27 +680,8 @@ class LimitClauseSegment(BaseSegment):
 ############################
 
 
-@exasol_dialect.segment(replace=True)
-class DropStatementSegment(BaseSegment):
-    """A `DROP` statement without any options."""
-
-    type = "drop_statement"
-    is_ddl = False
-    is_dml = False
-    is_dql = False
-    is_dcl = True
-    match_grammar = StartsWith("DROP")
-    parse_grammar = OneOf(
-        Ref("DropWithouOptionsStatementSegment"),
-        Ref("DropCascadeStatementSegment"),
-        Ref("DropCascadeRestrictStatementSegment"),
-        Ref("DropSchemaStatementSegment"),
-        Ref("DropTableStatementSegment"),
-    )
-
-
 @exasol_dialect.segment()
-class DropWithouOptionsStatementSegment(BaseSegment):
+class DropWithoutOptionsStatementSegment(BaseSegment):
```

Comment:
This isn't your fault but if you could add in the doc links for the statements covered by this it would be really helpful 😄

Reply:
We could probably split these segments out too but they're super niche so let's just leave as is

Reply:
✔

I would prefer to touch the exasol dialect as little as possible so will definitely defer this one to someone else